plugins/MySQL.cpp

```diff
@@ -297,6 +306,11 @@ void BedrockPlugin_MySQL::onPortRecv(STCPManager::Socket* s, SData& request) {
                 s->send(MySQLPacket::serializeQueryResponse(packet.sequenceID, result));
             } else if (SStartsWith(SToUpper(query), "SET ") || SStartsWith(SToUpper(query), "USE ")) {
                 // Ignore
+                SINFO("Responding OK to SET or USE query.");
+                s->send(MySQLPacket::serializeOK(packet.sequenceID));
+            } else if (SStartsWith(SToUpper(query), "ROLLBACK ") || SStartsWith(SToUpper(query), "ROLLBACK;")) {
```

Comment:
```
SStartsWith(SToUpper(query), "ROLLBACK")
```
doesn't work for both cases? 
Also these 2 else blocks being the same, perhaps we can collapse them for now?

Reply:
Also, shouldn't we trim(query) at the top?